docs/documentation/upgrading/topics/keycloak/changes-24_0_0.adoc

```diff
@@ -48,6 +48,36 @@ It is no longer allowed to have the same feature in both the `--features` and `-
 The usage of unversioned feature names, e.g. `docker`, in the `--features` list will allow for the most supported / latest feature version to be enabled for you. 
 If you need more predictable behavior across releases, reference the particular version you want instead, e.g. `docker:v1`.
 
+= User Profile Changes
+
+== Default validations
+
+Default user profile configuration comes with set of default validations for the basic predefined fields. Those validations were not present in previous versions when `declarative-user-profile` feature was disabled by default.
+If you have issues due the backwards compatibility, you can change the default validators according to your needs. The default validators are as follows:
+
+* All of `username`, `email`, `firstName` and `lastName` attributes have maximum length 255 characters. These validations were indirectly present in previous versions as well
+due there is DB constraint on the table `USER_ENTITY` for those fields with maximum length of 255 characters. However when using user storage providers, it might be possible before to use longer values.
```

Comment:
```suggestion
due there is a database constraint on the table `USER_ENTITY` for those fields with a maximum length of 255 characters. However when using user storage providers, it might be possible before to use longer values.
```